helpers/voting/getVoteMetaData.ts

```diff
@@ -27,8 +27,8 @@ export function getVoteMetaData(
       umipDataFromContentful?.description ??
       "No description was found for this UMIP.";
     const umipUrl = umipDataFromContentful?.umipLink;
-    const umipNumber = getUmipNumber(decodedIdentifier);
-    const links = makeVoteLinks(transactionHash, umipNumber);
+    const umipNumber = umipDataFromContentful?.number;
```

Comment:
For UMA governance, we only trust the information that comes from contentful. That data has the umip number and the url for the umip on github.